plugins/node/opentelemetry-instrumentation-pg/test/pg-pool.test.ts

```diff
@@ -269,6 +269,24 @@ describe('pg-pool', () => {
         assert.strictEqual(resNoPromise, undefined, 'No promise is returned');
       });
     });
+
+    it('should not generate traces when requireParentSpan=true is specified', async () => {
+      // The pool gets shared between tests. We need to create a separate one
+      // to test cold start, which can cause nested spans
+      const newPool = new pgPool(CONFIG);
+      create({
+        requireParentSpan: true,
+      });
+      memoryExporter.reset();
```

Comment:
is it needed?
looks like already happens at [`afterEach`](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/ff109b77928cc9a139a21c63d6b54399bb017fa4/plugins/node/opentelemetry-instrumentation-pg/test/pg-pool.test.ts#L171)

Reply:
You're right, thanks for the catch. I just removed this.